packages/agreement-process/src/routers/AgreementRouter.ts

```diff
@@ -265,12 +265,16 @@ const agreementRouter = (
       const ctx = fromAppContext(req.ctx);
 
       try {
-        validateAuthorization(ctx, [ADMIN_ROLE]);
+        validateAuthorization(ctx, [ADMIN_ROLE, M2M_ADMIN_ROLE]);
+
+        const { data: agreement, metadata } =
+          await agreementService.suspendAgreement(
```

Comment:
Leaving this comment in the last agreement m2m PR, but it's also related to previous requests:
In purposes/tenants/ecc, we aligned all error messages related to authorization/tenant not allowed to say "Tenant" instead of "Organization".
(For example: https://github.com/pagopa/interop-be-monorepo/pull/1870/files#diff-8f9bc27d0d93dc1544ddeb5a0ab6e8882ac5df2cc27da479d989eb133b80dca7R111)

I would also do the same for all agreement errors. I don't think it's necessary to change the function names of the validator, it's sufficient to change the error message and title